SPARCore/GADRemoteModel.m

```diff
@@ -0,0 +1,42 @@
+#import "GADRemoteModel.h"
+
+@implementation GADRemoteModel
+
+//static NSString *apiURL = @"https://g2j7qs2xs7.execute-api.us-west-2.amazonaws.com/devstable/publications";
```

Comment:
Remove if no longer needed. Eventually, we should probably move the `BASE_URL` and other relevant prefixes to this class. You could do this now or in a future PR.